library/alloc/src/raw_vec.rs

```diff
@@ -18,7 +18,7 @@ use crate::collections::TryReserveError::{self, *};
 #[cfg(test)]
 mod tests;
 
-#[cfg(not(no_global_oom_handling))]
+#[allow(dead_code)]
```

Comment:
Is this needed? `AllocInit` isn't dead code, and I don't get any warnings when I compile without this.

Reply:
Good catch!

Reply:
@Smittyvb It was in fact unused with `--cfg no_global_oom_handling`, but I just added some more `raw_vec` methods to fix that.